packages/expo-av/.npmignore

```diff
@@ -2,3 +2,4 @@
 babel.config.js
 __tests__
 __mocks__
+android/src/test/
```

Comment:
This needs to be added to expo-module-scripts – as you can see in the first line, this file is auto-generated 😉